packages/governance/src/types.js

```diff
@@ -490,11 +497,18 @@
  */
 
 /**
- * @typedef {Object} LimitedCreatorFacet
+ * @typedef {{
+ *   getGovernedApis: () => unknown
+ * }} GovernableCreatorFacet
+ */
+
+/**
+ * @template {GovernableCreatorFacet} [CF=any]
```

Comment:
If this isn't too onerous:
```suggestion
 * @template {GovernableCreatorFacet} [CF=unknown]
```

Reply:
When I do that, I get *Type 'unknown' does not satisfy the constraint 'GovernableCreatorFacet'*.